src/features/groups/list/index.styled.ts

```diff
@@ -1,12 +1,29 @@
 import styled from 'styled-components';
-import { PageContentWrapper, Row } from '../../../components/Containers';
+import { Column, PageContentWrapper, Row } from '../../../components/Containers';
 
-export const ContentWrapper = styled(PageContentWrapper)`
+export const ContentWrapper = styled(Column)`
+  position: relative;
+  overflow:hidden;
+  height: calc(100vh - 80px);
+`;
+
+export const GroupsListWrapper = styled(PageContentWrapper)`
   padding-top: 40px;
   flex: 1;
   gap: 30px;
+  z-index: 0;
 `;
 
 export const FilterTabsWrapper = styled(Row)`
   padding: 0 1rem;
 `;
+
+export const FilterTabItemsWrapper = styled(Column)`
+  overflow: auto;
+  height: calc(100vh - 11.2rem);
+  padding-bottom: 2rem;
```

Comment:
What the magic 11.2rem here?
Did you decide to use `rem` everywhere or just here? Don't we need to unify this approach over all project?